### PR TITLE
210 enhancement update examples with new data source and resource name

### LINKED ISSUE #210: [Enhancement]: Update examples with new data source and resource name
### Description

Current examples of the data source and resource are using old name. Need to update accordingly.

### Affected Resource(s) and/or Data Source(s)

_No response_

### Potential Terraform Configuration

_No response_

### References

_No response_

### Would you like to implement a fix?

None

### PR DESCRIPTION
**Pull request by chuyich**
fix #210

### CODE REVIEW

**Review by suhasbshekar — APPROVED** [review]

File: docs/data-sources/protocols_cifs_local_group_data_source.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
-page_title: "netapp-ontap_protocols_cifs_local_group_data_source Data Source - terraform-provider-netapp-ontap"
```
Comment (carchi8py) [inline docs/data-sources/protocols_cifs_local_group_data_source.md:3]: Protocols need to be removed

File: docs/data-sources/protocols_cifs_local_group_data_source.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Retrieves protocols cifs local group configuration
 
 ## Example Usage
 ```terraform
-data "netapp-ontap_protocols_cifs_local_group_data_source" "local_group" {
```
Comment (carchi8py) [inline docs/data-sources/protocols_cifs_local_group_data_source.md:15]: Protocols need to be removed

File: docs/resources/storage_aggregate_resource.md [hunk 3]
```diff
@@ -26,7 +26,7 @@ Create/Modify/Delete an aggregate resource
 
 ```terraform
 # Create an aggregate
-resource "netapp-ontap_storage_aggregate_resource" "example" {
+resource "netapp-ontap_storage_aggregate" "example" {
```
Comment (carchi8py) [inline docs/resources/storage_aggregate_resource.md]: Storage need to be removed.

**Review by carchi8py — CHANGES_REQUESTED** [review]

**Comment by carchi8py** [comment]
Protocols and storage need to be removed (we shorted those names.

**Comment by chuyich** [comment]
@carchi8py Do those shorted names also applied to both data source and resource? Then the directories of the examples need to be updated accordingly.

**Review by carchi8py — APPROVED** [review]

**Pull request merged**